packages/data-store/src/did-discovery-provider.ts

```diff
@@ -0,0 +1,31 @@
+import { IAgentContext } from '@veramo/core'
+import { IDataStoreORM } from './data-store-orm'
+import { AbstractDidDiscoveryProvider, IDIDDiscoverMatch, IDIDDiscoveryDiscoverDidArgs } from '@veramo/did-discovery'
+
+export class ProfileDiscoveryProvider implements AbstractDidDiscoveryProvider {
```

Comment:
I think this implementation should be done in the `did-discovery` package

Reply:
At the moment `data-store` has these deps:

```
    "@veramo/core": "^1.2.0",
    "@veramo/did-discovery": "^1.2.0",
    "@veramo/did-manager": "^1.2.0",
    "@veramo/key-manager": "^1.2.0",
 ```

This is because this package has DB specific implementations of
- `DIDStore` (`did-manager`)
- `KeyStore` (`key-manager`)
- `DiscoveryProvider` (`did-discovery`)

I was simply continuing previously established pattern